sqlxmq_macros/src/lib.rs

```diff
@@ -231,6 +231,11 @@ pub fn job(attr: TokenStream, item: TokenStream) -> TokenStream {
         }
     }
 
+    let outer_docs = inner_fn
+        .attrs
+        .iter()
+        .filter(|attr| attr.path.is_ident(&format_ident!("doc")));
```

Comment:
`is_ident` is generic, so you can just write `is_ident("doc")`

Reply:
Thanks